rdr_service/dao/study_nph_dao.py

```diff
@@ -156,11 +157,12 @@ def timepoint_exist(order: Namespace, visit_type: StudyCategory, session) -> Tup
             if result:
                 return True, result
         return False, None
-
+    @lru_cache(maxsize=128, typed=False)
```

Comment:
Not sure why we are adding these, I understand the utilization of LRU for caching but I would opt to remove these.

Reply:
That `decorator` is used cache the return value of `get_study_category` to avoid redundant Network I/O calls to the database to fetch the same study category object if it is needed in more than one instance.

Reply:
Yes I know what the decorator is but this isnt going to fix the issues that we are having with the API latency.